proto/eth/v1/beacon_chain.proto

```diff
@@ -20,7 +20,6 @@ import "google/protobuf/descriptor.proto";
 import "proto/eth/ext/options.proto";
 import "proto/eth/v1/attestation.proto";
 import "proto/eth/v1/beacon_block.proto";
```

Comment:
linting is saying this is not needed, ( a few of the imports) we can take care of it in a separate PR.